tests/Unit/CssInlinerTest.php

```diff
@@ -3140,6 +3140,20 @@ public function inlineCssNotInDebugModeIgnoresOnlyInvalidExcludedSelector(): voi
         self::assertStringContainsString('<p class="z"></p>', $result);
     }
 
+    /**
+     * @test
+     */
+    public function addExcludedCssSelector(): void
```

Comment:
Please rename the test so that the test name is a sentence with a statement about the expected behavior: https://speakerdeck.com/oliverklee/test-driven-development-with-phpunit-1?slide=49

Also, please add blank lines between the test phases (setup, execute, verify) to make the structure of this test more obvious to the reader: https://speakerdeck.com/oliverklee/test-driven-development-with-phpunit-1?slide=26

Reply:
This has been fixed.